main/src/library/BigInt.flix

```diff
@@ -19,26 +19,31 @@ namespace BigInt {
     ///
```

Comment:
Just a quick comment: For arbitrary-sized arithmetic the operations are never constant size nor space. I suggest we simply leave them un-annotated for now.

Reply:
Roger that.